uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/Stats/StatsGathererTest.php

```diff
@@ -9,7 +9,6 @@
 use CRM_HRLeaveAndAbsences_Test_Fabricator_LeaveRequest as LeaveRequestFabricator;
 use CRM_HRCore_Test_Fabricator_Contact as ContactFabricator;
 use CRM_HRCore_Test_Fabricator_UFMatch as UFMatchFabricator;
```

Comment:
Have you run the tests locally? It looks like you just removed the `use` statement while the test that uses it is still the same